day_6/exercises/practice.rb

```diff
@@ -0,0 +1,29 @@
+class Practice
+  attr_accessor :name, :height, :weight
+
+  def initialize(n, h, w)
```

Comment:
I'd say general rule of thumb, always avoid one letter variables or parameters. Be verbose and clear. I want to know what is what. These params don't tell me anything.